### DISCUSSION TITLE
Option to Capture Webhook Data

### DISCUSSION THREAD

**Discussion by cboecking** [PostgREST/postgrest#3645] [Q&A] [answered]
Thanks team for all that you do! I have a quick question about capturing webhook data.

Is there a way I can configure postgrest to receive/capture webhook data from an external system? I believe the answer is no. I believe I would need to write this tool myself and host it next to postgrest on a separate port/ip.

Here are are the details of the to-be solution:
* There are times when you need to know when an external system even occurs (example: webhook from github)
* I have no control over the webhook's payload. It is often large and complex.
* My goal is to simply persist the webook for downstream processing.
* In concept, this solution would behave similarly to webhook.site

Thanks for your time and consideration!
Chuck

**Comment by wolfgangwalther** [comment, marked as answer]
You could look into RPCs with a single unnamed json argument: https://postgrest.org/en/v12/references/api/functions.html#functions-with-a-single-unnamed-json-parameter

This should let you deal with the differing payload easily.

**Reply by cboecking** [reply]
Hi @wolfgangwalther - thank you! I am glad I asked.
